scripts/select_appointments.sql

```diff
@@ -1 +1,18 @@
 -- retrieve appointment information
+SELECT p.given_name, p.surname, a.time, a.date_of_appointment
+FROM patients p
+INNER JOIN appointments a
+  ON a.patient_id = p.id
+INNER JOIN doctors d
+  ON a.doctor_id = d.id
+WHERE a.doctor_id = 6
```

Comment:
We're currently looking at the `doctors` table, so we can't look it up by a property on `a`. How could we indicate that we want appointments that belong to Dr. Hubbard